docs/src/literate-howto/postprocessing.jl

```diff
@@ -46,7 +46,7 @@ include("../tutorials/heat_equation.jl");
 # Next we define a function that computes the heat flux for each integration point in the domain.
 # Fourier's law is adopted, where the conductivity tensor is assumed to be isotropic with unit
 # conductivity ``\lambda = 1 ⇒ q = - \nabla u``, where ``u`` is the temperature.
-function compute_heat_fluxes(cellvalues::CellValues{<:ScalarInterpolation}, dh::DofHandler, a::AbstractVector{T}) where T
+function compute_heat_fluxes(cellvalues::CellValues, dh::DofHandler, a::AbstractVector{T}) where T
```

Comment:
If required as a type signature, my suggestions would be
* Change the parameterization to `CellValues{IP, FV<:FunctionValues{IP}, ...}`
  This would restrict to having `FunctionValues` and preventing someone from extending with a custom `FunctionValues` type (not sure if that will be useful to do though)
* Define `const CellVectorValues = CellValues{<:FunctionValues{<:VectorInterpolation}}` and similar for `CellScalarValues`, and the `FaceValues` equivalents

But I couldn't find a case when this is required for dispatch, only for ensuring correct inputs. For those cases, it would be possible with the new interface to use, e.g.,
`@assert Ferrite.get_function_interpolation isa VectorInterpolation`